tests/integration_tests/modules/test_ssh_keys_provided.py

```diff
@@ -138,7 +139,11 @@ def test_ssh_provided_keys(self, config_path, expected_out, class_client):
         assert expected_out in out
 
     @pytest.mark.parametrize(
-        "expected_out", ("HostCertificate /etc/ssh/ssh_host_rsa_key-cert.pub")
+        "expected_out",
```

Comment:
I know you didn't add this, but I don't understand the point of the parameterization here. Can we remove it because as written it's running two separate tests with the same input and validating one line of output each time. Can we just validate both lines at the same time?